appveyor.yml

```diff
@@ -9,7 +9,7 @@ platform:
   - x64
 
 environment:
-  QT5: C:\Qt\5.12.2\msvc2017_64
+  QT5: C:\Qt\5.13\msvc2017_64
```

Comment:
please also update INSTALL accordingly specifying the minimum version required for Qt. Is it 5.13 now?

Reply:
yes, you're right.
`Format_Grayscale16` needs Qt 5.13, `Format_RGBA64`, `Format_RGBX64` needs Qt 5.12. It could be ifdef for compatibility if needed, but yes for now it requires 5.13.